test/integration/adapters/postmark_test.exs

```diff
@@ -5,23 +5,42 @@ defmodule Swoosh.Integration.Adapters.PostmarkTest do
 
   @moduletag integration: true
 
-  setup_all do
-    config = [api_key: System.get_env("POSTMARK_API_KEY"), domain: System.get_env("POSTMARK_DOMAIN")]
-    {:ok, config: config}
-  end
+  @config [
```

Comment:
This changes the behaviour as it will be grabbing the environment variables at compile-time rather than runtime, that can/will cause problem in some environments.